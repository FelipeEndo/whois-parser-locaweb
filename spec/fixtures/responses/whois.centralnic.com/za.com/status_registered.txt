Domain Name: SIYENZA.ZA.COM
Registry Domain ID: D333077-CNIC
Registrar WHOIS Server: whois.megaweb.co.za
Registrar URL: http://www.megaweb.co.za/
Updated Date: 2019-01-30T08:43:57.0Z
Creation Date: 2004-11-17T11:47:29.0Z
Registry Expiry Date: 2022-11-17T23:59:59.0Z
Registrar: Megaweb Internet Services
Registrar IANA ID:
Domain Status: ok https://icann.org/epp#ok
Domain Status: autoRenewPeriod https://icann.org/epp#autoRenewPeriod
Registrant Email: https://whois.centralnic.com/contact/siyenza.za.com/registrant
Admin Email: https://whois.centralnic.com/contact/siyenza.za.com/admin
Tech Email: https://whois.centralnic.com/contact/siyenza.za.com/tech
Name Server: NS1A.YOUR-SERVER.CO.ZA
Name Server: NSA.SECOND-NS.CO.ZA
DNSSEC: unsigned
Registrar Abuse Contact Email: dns-admin@megaweb.co.za
Registrar Abuse Contact Phone: +27.27114851984
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2021-12-15T17:21:34.0Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

>>> IMPORTANT INFORMATION ABOUT THE DEPLOYMENT OF RDAP: please visit
https://www.centralnic.com/support/rdap <<<

The Whois and RDAP services are provided by CentralNic, and contain
information pertaining to Internet domain names registered by our
our customers. By using this service you are agreeing (1) not to use any
information presented here for any purpose other than determining
ownership of domain names, (2) not to store or reproduce this data in
any way, (3) not to use any high-volume, automated, electronic processes
to obtain data from this service. Abuse of this service is monitored and
actions in contravention of these terms will result in being permanently
blacklisted. All data is (c) CentralNic Ltd (https://www.centralnic.com)

Access to the Whois and RDAP services is rate limited. For more
information, visit https://registrar-console.centralnic.com/pub/whois_guidance.
