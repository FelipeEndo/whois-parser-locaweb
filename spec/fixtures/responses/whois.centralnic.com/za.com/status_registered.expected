#disclaimer
  %s == "The Whois and RDAP services are provided by CentralNic, and contain information pertaining to Internet domain names registered by our our customers. By using this service you are agreeing (1) not to use any information presented here for any purpose other than determining ownership of domain names, (2) not to store or reproduce this data in any way, (3) not to use any high-volume, automated, electronic processes to obtain data from this service. Abuse of this service is monitored and actions in contravention of these terms will result in being permanently blacklisted. All data is (c) CentralNic Ltd (https://www.centralnic.com) Access to the Whois and RDAP services is rate limited. For more information, visit https://registrar-console.centralnic.com/pub/whois_guidance."


#domain
  %s == "siyenza.za.com"

#domain_id
  %s == "D333077-CNIC"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2004-11-17 11:47:29 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2019-01-30 08:43:57 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2022-11-17 23:59:59 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == ""
  %s.name         == "Megaweb Internet Services"
  %s.url          == "http://www.megaweb.co.za/"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].email         == "https://whois.centralnic.com/contact/siyenza.za.com/registrant"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].email         == "https://whois.centralnic.com/contact/siyenza.za.com/admin"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].email         == "https://whois.centralnic.com/contact/siyenza.za.com/tech"


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1a.your-server.co.za"
  %s[1] %CLASS{nameserver}
  %s[1].name == "nsa.second-ns.co.za"
